tests/FVTTests/fileParameterView/testFileParameterView.ts

```diff
@@ -76,6 +63,12 @@ describe('JES explorer spool file in url query (explorer-jes/#/viewer)', functio
  
     });
 
+    after('Close out', async () => {
```

Comment:
This is a duplicate of the after function on line 43

Reply:
removed

Reply:
removed